coding-concepts.js

```diff
@@ -20,24 +20,24 @@ const greeter = (name) => {
 }
 // console.log(greeter("LEARN Student"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer:this will log Hello, LEARN Student
+// b) Verify and explain: name is a pareamter that can have values passed thru . in this case the value was learn stuedents and plus the string hello (all returned using string interpulation)
```

Comment:
Make sure you uncomment the console log and see the output. This will result in undefined. Why?